style/main.css

```diff
@@ -0,0 +1,795 @@
+  }
+
+  .nav-main {
+    display: none;
+  }
+}
+
+.performing-artists .artist-card {
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+}
+
+@media screen and (max-width: 500px) {
+  .performing-artists .artist-card {
+    grid-template-columns: 1fr 2fr;
+    align-items: center;
+  }
+}
+
+.artist-image {
+  width: 160px;
+  height: 7rem;
+  position: relative;
+  font-size: 14px;
+  font-style: normal;
+  line-height: 22px;
+}
+
+@media screen and (max-width: 500px) {
+  .speakers_list_item_img {
+    width: 130px;
+    height: 6rem;
+  }
+}
+
+@media screen and (max-width: 500px) {
+  .speakers_list_item_img {
+    width: 120px;
+  }
+}
+
+.artist-bio .thin-line {
+  margin: 0.5rem 0;
+  border-bottom: 2px solid #d3d3d3;
+}
+
+.artist-bio h4 {
+  margin-top: 5px;
+}
+
+.speaker_list_item_content_bio {
+  font-size: 12px;
+  font-style: normal;
+  line-height: 14px;
+  color: #ec5242;
+  font-family: "Lato", sans-serif;
+}
+
+.program-list {
+  list-style: none;
+  z-index: 2;
+}
+
+.program-list li {
+  background: rgba(78, 81, 91, 0.5);
+  border-radius: 3px;
+  margin-top: 0.5rem;
+  padding: 0.75rem;
+  display: flex;
+  align-items: center;
+  padding-top: 20%;
+  transition: 0.8s ease-in-out;
+}
+
+.program-list li:hover {
+  border: 1px solid #fff;
+}
+
+.program-list li i {
+  min-width: 45px;
+  height: 45px;
+  background-size: 95%;
+  background-repeat: no-repeat;
+}
+
+.program-list li h5 {
+  font-family: "cocogoose", sans-serif;
+  font-weight: 200;
+  color: #ec5242;
+  padding: 0 0.75rem;
+  min-width: 100px;
+}
+
+.program-list li p {
+  font-size: 14px;
+  font-style: normal;
+  line-height: 16px;
+}
+
+.program-list .shed-icon {
+  background-image: url(../icons/shed-icon.png);
+}
+
+.program-list .master-class {
+  background-image: url(../icons/master-classs.jpg);
+}
+
+.program-list .Meet-up-icon {
+  background-image: url(../icons/Meet-up-icon.gif);
+}
+
+.program-list .exhibition-icon {
+  background-image: url(../icons/Drum-exhibition.svg);
+  background-size: 105%;
+}
+
+.program-list .solo-icon {
+  background-image: url(../icons/drum-solo.svg);
+}
+
+.artist-bio p:nth-child(4) {
+  color: rgb(39, 42, 49);
+}
+
+.copyright {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: goldenrod;
+  padding: 1rem 0;
+  width: 100%;
+}
+
+.copyright img {
+  width: 80px;
+}
+
+.copyright h6 {
+  padding: 1rem;
+  font-size: 1rem;
+  color: rgb(39, 42, 49);
+  max-width: 230px;
+  line-height: 15px;
+}
+
+.footer {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-color: rgb(39, 42, 49);
+  padding: 1rem;
+}
+
+.footer h4 {
+  color: #d3d3d3;
+}
+
+.home-footer .brand-support {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.home-footer .brand-support img {
+  min-height: 46px;
+  flex: 1 0 auto;
+  width: 25%;
+  padding: 0.5rem 0.5rem;
+}
+
```

Comment:
- Please, kindly add a filter property to the partners to match the design template below:


*yours partners*
![add](https://user-images.githubusercontent.com/59397159/175294499-229fbb50-40da-4fab-b98e-9cedfd9f6dee.png)





*design partners*
![opacity__](https://user-images.githubusercontent.com/59397159/175101159-c339e369-03e7-49f2-a361-a260e988dda3.png)



- To achieve that, add this `filter: grayscale(100%) contrast(30%)`.
*Note: this is just one way to get the result*